decidim-proposals/config/locales/en.yml

```diff
@@ -637,10 +637,6 @@ en:
           proposal_in_evaluation_reason: This proposal is being evaluated
           proposal_rejected_reason: 'This proposal has been rejected because:'
           report: Report
-          sign_in: Sign in
-          sign_in_or_up: "%{in} or %{up} to participate"
-          sign_in_to_participate: "%{in} to participate"
```

Comment:
Before, if there was no user, instead of the follow button, a message was shown: "Sign in or sign up to participate". But as discussed here https://github.com/decidim/decidim/issues/3752#issuecomment-441998111 , the follow button should always show, so these locales are no longer used.